### ISSUE TITLE
plot of mixed_energy in simulation

### ISSUE THREAD

**Issue by NOrangeeroli** [bytedance/jaqmc#2]
Hi, I tried this code on N2, the vmc model is ferminet. I got the following results when plot mixed energy during dmc simulation:
![image](https://user-images.githubusercontent.com/24457676/208254147-e27755ab-51be-4f41-9382-d70702e1f557.png)
the y axis is mixed_energy-exact_energy. I set mixed_estimator_num_steps= 2000
We find the variance of mixed_energy drops suddenly at 2000th step. Is this an expected behavior? 
And after 2000th steps, the variance is lower than before 2000th step but still pretty high (it swings in a range of 8 mHartrees). How is the final result reported in the paper calculated? I mean if it swings so much we can not use the mixed_energy of the last step for report.
Thanks in advance.

**Comment by WeiluoRen-ByteDance** [comment]
Hey @NOrangeeroli,

Thanks for trying out JaQMC :)

The large fluctuation in the beginning is as expected and is due to the change of the time window of calculating the mixed energy. (Let me know if you want to know DMC-related technical details and I can explain more)

You are quite right, even at the convergence, the fluctuation of mixed_energy is still too large, which is because the auto-correlation of averaged energy calculated from each step is quite large and 2000 steps is simply too small. However if we set the mixed energy time window to be large enough, there's a potential numerical issue. So we made a compromise that the mixed energy is mainly for updating the walkers' weight and we do reblock analysis on the averaged energies to get the reported results. Another major reason for doing reblock analysis on averaged energy for report purpose is that in this way it's easier to get the error bar of our energy result. We run DMC for hundreds of thousands of steps and use the last 90% steps in our reblock analysis, which means we involves tens of thousands of steps in the reblock analyis, significantly greater than the mixed energy time window. We mentioned how we get our energy results in section IV.D in our DMC paper.

Let me know if it helps. Happy to explain more if you are interested :)
